### ISSUE TITLE
some questions about the code

### ISSUE THREAD

**Issue by Guibeen** [LIN-WANQIU/ADI17#1]
Hi, professors. These days I’m reading the code in this repository, beacause I want to repeat this work. I got some problems that I can't figure out, and I think I need your help:

1. In line 65, train_models.py, `train_dataloader = m5DataLoader(train_data,batch_size=10)` : I cannot find where the variable `train_data` is declared or initialized.

2. In line 67, the same file above, `acc_data = m5Dataset('data/acc2/')` : how is the dir 'acc2' created and what content is in it?

I'm looking forward to your reply, thanks.

**Comment by Guibeen** [comment]
for the 2nd question above, I think it may be produced by executing prep_acc.sh. Is it ?

**Comment by LIN-WANQIU** [comment]
Hi,
Thank you for your email. Sorry for not reply you in time since I'm busy in the working day.&nbsp;
In response to your first question, the variable "train_data" may have been accidentally deleted when the file was uploaded. Now I have corrected this error. This variable is actually similar to the "acc_data" variable in the second question you asked. These two variables are the paths of the training set and the test set respectively. You have to prepare these data before training the model.



Regards,
Wanqiu




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "LIN-WANQIU/ADI17"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月10日(星期二) 下午5:05
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[LIN-WANQIU/ADI17] some questions about the code (#1)





 
Hi, professors. These days I’m reading the code in this repository, beacause I want to repeat this work. I got some problems that I can't figure out, and I think I need your help:
  
 
In line 65, train_models.py, train_dataloader = m5DataLoader(train_data,batch_size=10) : I cannot find where the variable train_data is declared or initialized.
 
 
 
In line 67, the same file above, acc_data = m5Dataset('data/acc2/') : how is the dir 'acc2' created and what content is in it?
 
  
I'm looking forward to your reply, thanks.
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Guibeen** [comment]
Hi, Prof. LIN. Thanks to you answer, now I've successfully started the training. Thank you very much!
